Week3/homework/3-step3.js

```diff
@@ -2,23 +2,39 @@
 
 // use a 'for' loop
 function repeatStringNumTimesWithFor(str, num) {
-  // add your code here
-  return str;
+  let newStr = str;
+  if (num > 0) {
+    for (let i = 2; i <= num; i++) {
+      str += newStr;
+    }
+    return str;
+  }
+  else { return str = "" }
```

Comment:
If the `if` block is executed it will be never consider the `else` block (because the `if` block ends in `return`).  Therefore line 12 might simply be: 

```js
return "";
```